### ISSUE TITLE
Ну как, на работу взяли?

### ISSUE THREAD

**Issue by lbarmen** [Angver/mirafoxTestCase#1]
Ну как, на работу взяли? :) прислали тоже это тестовое задание)

**Comment by Angver** [comment]
@lbarmen нет, не взяли. Что-то им не понравилось( Как известно, выполненные тестовые задания они не комментируют

**Comment by lbarmen** [comment]
@Angver ясно), надо осторожнее выбирать такие вакансии, нынче много таких контор, которые хотят выполненное задание, а потом даже не утруждают себя ответить, что именно им не подошло.

**Comment by mosinve** [comment]
Тоже делал такое задание для них и тоже не взяли :)
